urnai/actions/chain_of_actions_base.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC
+
+from urnai.actions.action_base import ActionBase
+
+
+class ChainOfActionsBase(ABC):
+
+	"""This class works as a sequence of actions"""
+	
+	action_list: list[ActionBase]
```

Comment:
Does this declaration work? I couldn't identify where the `list` is declared.

Reply:
It works for python 3.9 or greater. Its the actual `list` type, i found it [here](https://stackoverflow.com/questions/64496437/python-list-type-declaration).